Description: Link the time module to librt
 FIXME: Back-story?

Forwarded: no

Index: b/Modules/Setup.dist
===================================================================
--- a/Modules/Setup.dist
+++ b/Modules/Setup.dist
@@ -118,7 +118,7 @@ _collections _collectionsmodule.c # Cont
 itertools itertoolsmodule.c    # Functions creating iterators for efficient looping
 atexit atexitmodule.c      # Register functions to be run at interpreter-shutdown
 _stat _stat.c			# stat.h interface
-time timemodule.c	# -lm # time operations and variables
+time timemodule.c -lrt	# -lm # time operations and variables
 
 # access to ISO C locale support
 _locale _localemodule.c  # -lintl
